mpcforces_extractor/api/db/database.py

```diff
@@ -202,35 +202,54 @@ def populate_mpcs(self, session):
         """
         Function to populate the database with MPCs
         """
-        for mpc_config in MPC_CONFIG:
-            if mpc_config.value not in MPC.config_2_id_2_instance:
-                continue
-            for mpc in MPC.config_2_id_2_instance[mpc_config.value].values():
-                mpc.get_part_id2force(None)
-                sub2part2force = mpc.get_subcase_id2part_id2force()
-
-                if mpc_config == MPC_CONFIG.RBE2:
-                    db_mpc = RBE2DBModel(
-                        id=mpc.element_id,
-                        config=mpc.mpc_config.name,  # Store enum as string
-                        master_node=mpc.master_node.id,
-                        nodes=",".join([str(node.id) for node in mpc.nodes]),
-                        part_id2nodes=mpc.part_id2node_ids,
-                        subcase_id2part_id2forces=sub2part2force,
-                    )
-                elif mpc_config == MPC_CONFIG.RBE3:
-                    db_mpc = RBE3DBModel(
-                        id=mpc.element_id,
-                        config=mpc.mpc_config.name,  # Store enum as string
-                        master_node=mpc.master_node.id,
-                        nodes=",".join([str(node.id) for node in mpc.nodes]),
-                        part_id2nodes=mpc.part_id2node_ids,
-                        subcase_id2part_id2forces=sub2part2force,
-                    )
-                else:
-                    raise ValueError(f"Unknown MPC config {mpc_config}")
-                # Add to the session
-                session.add(db_mpc)
+        for mpc in MPC.all_instances:
+            mpc.get_part_id2force(None)
+            sub2part2force = mpc.get_subcase_id2part_id2force()
+            # stresss
+            sub2part2axial_radial_forces = {}
+            for sub in Subcase.subcases:
+                sub2part2axial_radial_forces[sub.subcase_id] = (
+                    mpc.get_part_id2axial_radial_forces(sub)
+                )
+            _, _, max_rad_force, _, _, max_axial_force = (
+                mpc.get_max_radial_and_axial_force()
+            )
```

Comment:
The part id and the subcase where the max force is occuring is not used and I am not quite sure if I really dont want to use the subcase id from the dropdown menu.

Reply:
No, I will leave it as it is - as this is more important than the dropdown.